base/applications/notepad/notepad.h

```diff
@@ -93,7 +92,7 @@ extern NOTEPAD_GLOBALS Globals;
 BOOL ReadText(HANDLE hFile, HLOCAL *phLocal, ENCODING *pencFile, EOLN *piEoln);
 BOOL WriteText(HANDLE hFile, LPCWSTR pszText, DWORD dwTextLen, ENCODING encFile, EOLN iEoln);
 
-void NOTEPAD_LoadSettingsFromRegistry(void);
+void NOTEPAD_LoadSettingsFromRegistry(WINDOWPLACEMENT*pWP);
```

Comment:
```suggestion
void NOTEPAD_LoadSettingsFromRegistry(PWINDOWPLACEMENT pWP);
```
(same style change in the other places)